File: navigate_browser.py
import subprocess
import json
import os

mavis_path = r"C:\Users\Pick\.mavis\bin\mavis.cmd"

# Navigate to Supabase
url = "https://supabase.com/dashboard/project/pexcvfhuvqrwrabpgkzi/auth/providers?method=github"

result = subprocess.run(
    [mavis_path, "browser", "tool", "navigate"],
    input=json.dumps({"url": url}),
    capture_output=True,
    text=True
)
print("STDOUT:", result.stdout)
print("STDERR:", result.stderr)

File: navigate_browser.js
const { spawn } = require('child_process');

const url = "https://supabase.com/dashboard/project/pexcvfhuvqrwrabpgkzi/auth/providers?method=github";

const proc = spawn("cmd", ["/c", "mavis.cmd", "browser", "tool", "navigate"], {
    cwd: "C:\\Users\\Pick\\.mavis\\bin",
    stdio: ['pipe', 'pipe', 'pipe']
});

proc.stdin.write(JSON.stringify({ url }));
proc.stdin.end();

proc.stdout.on('data', (data) => process.stdout.write(data));
proc.stderr.on('data', (data) => process.stderr.write(data));
